convert2rhel/unit_tests/actions/pre_ponr_changes/backup_system_test.py

```diff
@@ -141,6 +141,7 @@ def test_backup_redhat_release_error_os_release_file(self, backup_redhat_release
             (
                 """     missing   d /usr/share/info/ed.info.gz
                 S.5.?..T. c     /etc/yum.repos.d/CentOS-Linux-AppStream.repo
+                S.5.?..T. c     /etc/yum.repos.d/CentOS-Linux-AppStream.repo
                 S.5.?..T.        /etc/yum.repos.d/CentOS-Linux-AppStream.repo
```

Comment:
I'd then remove the non-c entry from the `rpm_va_output` because I'd expect that two entries for the same file would be of the same type.
```suggestion
                S.5.?..T. c     /etc/yum.repos.d/CentOS-Linux-AppStream.repo
```
And then remove the following from the `expected`:
```
                    {
                        "status": "S5T",
                        "file_type": None,
                        "path": "/etc/yum.repos.d/CentOS-Linux-AppStream.repo",
                    },
```